main/java/dao/UserDao.java

```diff
@@ -0,0 +1,38 @@
+package dao;
+
+import db.DataBaceConnetcion;
+import model.UserModel;
+
+import javax.persistence.EntityManager;
+import javax.persistence.EntityManagerFactory;
+import javax.persistence.Query;
+import java.util.Iterator;
+import java.util.List;
+
+public class UserDao {
+   private EntityManagerFactory connection;
+
+    public UserDao() {
+        connection = DataBaceConnetcion.getConnection();
+    }
+
+    public void addUser(UserModel user) {
+        EntityManager entityManager = connection.createEntityManager();
+        entityManager.getTransaction().begin();
+        entityManager.persist(user);
+        entityManager.getTransaction().commit();
+    }
+
+    public UserModel checkIsPresent(String login) {
+        Query select_u_from_userModel_u = connection.createEntityManager().createQuery("select u from UserModel u");

```

Comment:
do we need get all list here? maybe exist other way to find only one result which equals login?